### PR TITLE
new 7.48.x branch

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

@mareknovotny @Ginxo locally (I suppose also on this PR) the script
for springboot_pr_job.groovy fails ./gradlew clean. I don't know why. It says:
https://gist.github.com/mbiarnes/392da371dcc703ca9a30b20d654dfaac

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/seed_job.groovy [hunk 1]
```diff
@@ -51,41 +51,33 @@ job("a-seed-job") {
...
+                    "job-dsls/jobs/**/seed_job.groovy")
             useScriptText(false)
             sandbox(false)
             ignoreExisting(false)
             ignoreMissingFiles(false)
             failOnMissingPlugin(true)
             unstableOnDeprecation(true)
             removedJobAction('IGNORE')
-            removedViewAction('DELETE')
+            removedViewAction('IGNORE')
```
Comment (mareknovotny) [inline job-dsls/jobs/seed_job.groovy:80]: why to change this attribute? just curious

**Comment by mareknovotny** [comment]
jenkins retest this please

**Comment by mbiarnes** [comment]
@Ginxo @mareknovotny https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/kie-jenkins-scripts.pr/156/console

File: job-dsls/jobs/seed_job.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/seed_job.groovy]: OK - we don't have views here - will fix this, mean will not change this

**Pull request closed without merging**